examples/03_connectivity/plot_atlas_comparison.py

```diff
@@ -127,16 +126,14 @@ def lag_correlation(time_series, lag):
 ##########################################################################
 # Iterate over fetched atlases to extract coordinates - probabilistic
 # -------------------------------------------------------------------
-from nilearn.maskers import NiftiMapsMasker
+from nilearn.maskers import MultiNiftiMapsMasker
 
 # create masker to extract functional data within atlas parcels
-masker = NiftiMapsMasker(maps_img=difumo.maps, standardize=True,
-                         memory='nilearn_cache')
+masker = MultiNiftiMapsMasker(maps_img=difumo.maps, standardize=True,
```

Comment:
Through a comment, make it explicit that you use `MultiNiftiMapsMasker`, so that people notice and understand why.